app/styles/globals.css

```
@@ -4,6 +4,42 @@
 @import './_utilities.css';
 @import 'tailwindcss';
 
+@font-face {
+  font-family: 'MaruBuri';
```

Stylelint (17.11.0)

[error] Expected no quotes around "MaruBuri" (font-family-name-quotes)

(font-family-name-quotes)